styles/main.css

```diff
@@ -54,13 +55,14 @@ header {
 }
 
 .photo {
-  background: url('../images/jaws.jpg') center 50%;
+  background: url('../images/myself.jpg') center 50%;
```

Comment:
Need to tweak this a bit, it's cutting you off oddly.

![image](https://user-images.githubusercontent.com/5775716/70648603-715e7f80-1c19-11ea-81f6-3b15b120c2ff.png)